dali/pipeline/pipeline.cc

```diff
@@ -384,9 +385,6 @@ int Pipeline::AddOperatorImpl(const OpSpec &const_spec, const std::string &inst_
                             ". Named argument inputs to operators must be CPU data nodes. "
                             "However, a GPU data node was provided."));
     }
-
-    if (device == "gpu" && separated_execution_)
-      SetupCPUInput(it, input_idx, &spec);
```

Comment:
I introduced a delay in Executor RunGPU (in different places) and tested it with the following code:
```Python
@params((2, 5), (5, 2))
def test_separated_queues(cpu_queue_depth, gpu_queue_depth):
    data = [np.array((10 * (i + 1)), dtype=np.float32) for i in range(10)]
    @pipeline_def(batch_size=1, num_threads=1, device_id=0)
    def pipe():
        inp = fn.external_source(data, batch=False, cycle=True) + 0
        img = types.Constant(np.zeros(shape=(1,1,3), dtype=np.uint8)).gpu()
        return fn.resize(img, resize_x=inp, resize_y=1)  # pass argument input to resize_x

    p = pipe(prefetch_queue_depth={"cpu_size":cpu_queue_depth, "gpu_size":gpu_queue_depth})
    p.build()
    for i in range(10):
        o, = p.run()
        assert o[0].shape()[1] == data[i]
```
The results were correct, so this seems not necessary now (I wonder if it ever was).